core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UniqueIDUserStoreManager.java

```diff
@@ -466,35 +453,22 @@ void updateUserListOfRoleWithID(String roleName, String[] deletedUserIDs, String
      * Add a group to the system.
      *
      * @param groupName     Group's display name.
-     * @param userIDList    Array of User IDs belongs to the group.
-     * @param permissions   Array of permissions of the group.
-     * @param isSharedGroup Whether the added group is a shared or not.
```

Comment:
Are we not supporting shared groups with the new implementation?

Reply:
we decided not to support for new APIs